Makefile

```
@@ -0,0 +1,115 @@
+# Makefile for trellis build artifacts
+# Use with pixi: pixi run make <target>
+
+.PHONY: all clean wheel bundles server-bundle desktop-bundle browser-bundle \
```

checkmake (0.2.2)

[warning] Missing required phony target "test"

(minphony)